### PR TITLE
Add echo for end of each build phase

### PR DESCRIPTION
**Pull request by richsalz**
It is hard to find out what failed when trolling Travis logs.  This makes it easier, you can just skip to /// and scroll back looking for error messages.
After reviewed, I'll do similar PR's for master and 1.0.2

### CODE REVIEW

**Comment by levitte** [comment]
If you grep for 'exited', you'll find the ends that you look for already.

**Comment by dot-asm** [comment]
Travis red cross is relevant. It complains about not knowing how to 'make echo'. As for searching for "exited". It's valid point, but many of them are not directly related, so you still kind of want "own" markers...

File: .travis.yml [hunk 1]
```diff
@@ -164,11 +164,14 @@ script:
           cd _build;
       fi
     - $make update;
+      echo '/// MAKE UPDATE END ///';
       git diff --quiet
     - if [ -n "$CHECKDOCS" ]; then
           $make doc-nits;
+          echo '/// MAKE DOC-NITS END ///';
       fi
     - $make
```
Comment (levitte) [inline .travis.yml]: You need to add a semi (;) at the end of this line.

**Comment by richsalz** [comment]
Added the semicolon and pushed an updated commit.

**Comment by dot-asm** [comment]
Do you know what would be more useful? If it resulted in unique hit per search. I mean how looking for say '///' gives thee hits, one in "starting this", then echo itself and finally "this exited". And what would be even more useful if first '///' would take you to first check-point, '////' - to second, etc. This way if you have travis log open in firefox, you'd be able to advance to next checkpoint by simply adding '/' to find dialogue...

**Comment by dot-asm** [comment]
> three hits

I mean `echo "\057\057\057"` would do the trick...

**Comment by levitte** [comment]
Ehhhh, I just realised I must disagree with this PR, as it will change useful exit codes.

```
$ (exit 1); echo foo; echo $?
foo
0
```

So, errr, -1

**Comment by dot-asm** [comment]
> as it will change useful exit codes.

Good catch! But it should still be possible to add markers as separate lines or if clauses. For example not 
```
    - if [ -n "$CHECKDOCS" ]; then
          $make doc-nits;
          echo "something"
      fi
```
but two ifs, second is specifically for echo marker...

**Comment by richsalz** [comment]
My use-case is that I do control-F and then /// in the browser to jump to the end of each section to see if there are any failures.  So I removed the trailing ///  Does this make sense?

**Comment by richsalz** [comment]
I could replace each of the added echo lines with this:
```
case $? in
0) echo '/// MAKE UPDATE OK' ;;
*) echo '/// MAKE UPDATE FAILED' ; /bin/false ;;
esac
```

But that seems too much.  I'll see if searching for "exited" works well enough for me.

**Pull request closed without merging**